### PR TITLE
#630/fl settings navbar layout

### PR DESCRIPTION
**Pull request by pranotoism**
### Detailed description of changes

...

### Recommendations on how to test the changes

...

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

#### Before

...

#### After

...

### CODE REVIEW

**Comment by ardiwine** [comment]
@pranotoism tolong di check checklist yang dibuat sama rika di description PR nya. pastiin itu semua sudah di kerjain, dan tolong di check masing2 boxnya kalau memang sudah .

**Comment by ardiwine** [comment]
@pranotoism component testing nya belum ada?

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

File: src/Components/Atoms/FAOutlinedInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { forwardRef, memo } from "react";
+import FormControl, { useFormControl } from "@mui/material/FormControl";
+import OutlinedInput from "@mui/material/OutlinedInput";
+import Box from "@mui/material/Box";
+import FormHelperText from "@mui/material/FormHelperText";
+import { IInput } from "./Input.interface";
+import { InputLabel } from "@mui/material";
```
[the remark this answers is not in the document]
Comment (pranotoism) [reply src/Components/Atoms/FAOutlinedInput/index.tsx:7]: done

File: src/Components/Organisms/FOHeader/FOHeader.interfaces.tsx [hunk 2]
```diff
@@ -14,4 +21,5 @@ export interface IFOHeader extends AppBarProps {
   menuIcon?: boolean;
   endAdornment?: ReactChild;
   startAdornment?: ReactChild;
+  open?: boolean;
```
[the remark this answers is not in the document]
Comment (pranotoism) [reply src/Components/Organisms/FOHeader/FOHeader.interfaces.tsx:24]: im gonna stick to the original as the MUI guide just to make it clean and simple

File: src/Components/Atoms/FAInput/FAInput.interfaces.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { InputProps } from "@mui/material";
+
+export interface IInput extends InputProps {
+  testID: `input-${string}`;
+  label?: string;
+  variant?: "standard" | "outlined" | "filled";
+  helperText?: ReactChild | JSX.Element | string;
+  focusedText?: ReactChild;
+  hook?: any;
+  error?: boolean;
```
[the remark this answers is not in the document]
Comment (pranotoism) [reply src/Components/Atoms/FAInput/FAInput.interfaces.tsx:11]: im gonna stick to the original as the MUI guide just to make it clean and simple

**Review by gilangdarmawan — APPROVED** [review]

**Pull request merged**